src/computed/seed.js

```diff
@@ -0,0 +1,65 @@
+import { computed, extendObservable } from 'mobx';
+
+const ComputedSeed = store => {
+  extendObservable(store, {
+    seedVerifyIndexes: computed(() => {
+      const { seedMnemonic: words } = store;
+      return words.length ? getSeedCheck(1, words.length, 3) : [];
+    }),
+    seedVerifyCopy: computed(() => {
+      const { seedVerifyIndexes } = store;
+      const c0 = formatOrdinal(seedVerifyIndexes[0]);
+      const c1 = formatOrdinal(seedVerifyIndexes[1]);
+      const c2 = formatOrdinal(seedVerifyIndexes[2]);
+      return `Type the ${c0}, ${c1}, and ${c2} words of your seed.`;
+    }),
+  });
+};
+
+/**
+ * Get an array of seed word indexes to verify
+ * @param  {number} min      The smallest possible index
+ * @param  {number} max      The largest possible index
+ * @param  {number} numWords The length of the array to generate
+ * @return {Array}           The array of indexes to verify
+ */
```

Comment:
I added some JSdocs here. We should probably do this for all new code being written. I've been slacking on that front up until now.